wolfssl/wolfcrypt/kyber.h

```diff
@@ -0,0 +1,228 @@
+/* kyber.h
```

Comment:
Do we need some stuff for 90S as well?

Reply:
I made the 90s a configuration option.
You get one or the other but not both.